procedure

```diff
@@ -0,0 +1,103 @@
+
+
+DROP PROCEDURE IF EXISTS cost_setter $$
+CREATE PROCEDURE cost_setter()
+
+BEGIN
+
+DECLARE total_sum  INTEGER;
+DECLARE order_count INTEGER;
+DECLARE itr INTEGER;
+DECLARE discount INTEGER;
+DECLARE pay INTEGER; 
+DECLARE discount_amout INTEGER;
+
+set order_count = (select count(*) from orders);
+set itr=1;
+
+while(itr <= order_count) do
```

Comment:
Why you need while loop, Its the matter of single order, why you have to update all the orders.